addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2759,6 +2762,10 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
             width: `${this.$target.innerWidth()}px`,
             height: `${this.$target.innerHeight()}px`,
         });
+
+        const $buttons = this.$overlayContent.find('div:last-child');
+        const topPosition = parseInt(this.$overlay.css('top')) - parseInt(this.$overlayContent.css('top')) + 'px';
+        $buttons.css('top', topPosition);
```

Comment:
All of this should be done as a fix in 13.1/2.

Reply:
> All of this should be done as a fix in 13.1/2.

ok => https://github.com/odoo/odoo/pull/48356